scripts/patrol-gui.lua

```diff
@@ -171,6 +186,18 @@ local function build_waypoint_player_input(i, waypoint)
         tags = {index = i},
       }
     }
+  elseif waypoint_type == "liftoff" then
+     local condition_info = waypoint.circuit_condition_info ---@cast condition_info -?
+     return {
+       {
+         type = "choose-elem-button",
+         style = "train_schedule_item_select_button",
+         elem_type = "space-location",
+         ["space-location"] = condition_info.elem,
+         handler = { [defines.events.on_gui_elem_changed] = PatrolGuiWaypoint.condition_elem_selected },
+         tags = { index = i },
+       }
+     }
```

Comment:
Please move this to the end of the `if/elseif` chain (i.e. line 218)

Reply:
and match style to following code (i.e. not all table entries on their own line)

Reply:
moved it.